composer.json

```diff
@@ -14,7 +14,7 @@
 		}
 	],
 	"require": {
-		"php": ">=5.3.1",
+		"php": ">=5.3.7",
```

Comment:
@dg OT: Current Debian stable Wheeze has 5.4.36. New stable Jessie should be out in March with 5.6.5.